wallet/grpc/server/src/lib.rs

```diff
@@ -83,8 +83,16 @@ impl Kaspawalletd for Service {
         Ok(Response::new(BroadcastResponse { tx_ids }))
     }
 
-    async fn broadcast_replacement(&self, _request: Request<BroadcastRequest>) -> Result<Response<BroadcastResponse>, Status> {
-        todo!();
+    async fn broadcast_replacement(&self, request: Request<BroadcastRequest>) -> Result<Response<BroadcastResponse>, Status> {
+        let request = request.into_inner();
+        let txs = deserialize_txs(request.transactions, request.is_domain, self.use_ecdsa())?;
+        let mut tx_ids: Vec<String> = Vec::with_capacity(txs.len());
+        for tx in txs {
+            let submit_transaction_replacement_response =
+                self.wallet().rpc_api().submit_transaction_replacement(tx).await.map_err(|e| Status::new(Code::Internal, e.to_string()))?;
+            tx_ids.push(submit_transaction_replacement_response.transaction_id.to_string());
+        }
```

Comment:
this logic should also be ported
https://github.com/kaspanet/kaspad/blob/4bb5bf25d3f2279ec2a61c3b4f7bb083b5f522b2/cmd/kaspawallet/daemon/server/broadcast_replacement.go#L50-L64

i mean only first tx should be sent as replacement